wallaby.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function(wallaby) {
```

Comment:
I don't want to get into a philosophical debate, but can we please remove this file? I don't want to support proprietary, closed-source software in an OSS project. It's fine that you use it locally, but it's not a requirement to develop for the library nor do I really want to promote them from this project.